L1Trigger/L1TMuonEndCap/python/simEmtfDigis_cfi.py

```diff
@@ -16,24 +16,32 @@
     # Configure by firmware version, which may be different than the default parameters in this file
     FWConfig = cms.bool(True),
 
+    # Era (options: 'Run2_2016', 'Run2_2017', 'Run2_2018')
+    Era = cms.string('Run2_2018'),
+
     # Input collections
     # Three options for CSCInput
     #   * 'simCscTriggerPrimitiveDigis','MPCSORTED' : simulated trigger primitives (LCTs) from re-emulating CSC digis
     #   * 'csctfDigis' : real trigger primitives as received by CSCTF (legacy trigger), available only in 2016 data
     #   * 'emtfStage2Digis' : real trigger primitives as received by EMTF, unpacked in EventFilter/L1TRawToDigi/
-    CSCInput  = cms.InputTag('simCscTriggerPrimitiveDigis','MPCSORTED'),
-    RPCInput  = cms.InputTag('simMuonRPCDigis'),
+    DTPhiInput = cms.InputTag('simTwinMuxDigis'),
+    DTThetaInput = cms.InputTag('simDtTriggerPrimitiveDigis'),
+    CSCInput = cms.InputTag('simCscTriggerPrimitiveDigis','MPCSORTED'),
+    CSCComparatorInput = cms.InputTag('simMuonCSCDigis','MuonCSCComparatorDigi'),
+    RPCInput = cms.InputTag('simMuonRPCDigis'),
+    RPCRecHitInput = cms.InputTag('rpcRecHits'),
```

Comment:
By quick look it seems that this change could have broken a unit test in `TauAnalysis/MCEmbeddingTools`, see https://github.com/cms-sw/cmssw/issues/29897.

Reply:
Sorry, I didn't realize this could be an issue. Yes, the rpcRecHits is not needed at the moment, and can be removed. Should I create a new pull request?

Reply:
> Should I create a new pull request?

Yes, thanks!

Reply:
> the rpcRecHits is not needed at the moment

On the other hand, if it becomes needed in the future, it needs to be discussed because that would break the test again. (`rpcRecHits` is part of reconstruction, which makes me wonder why L1 simulation should depend on reconstruction, but I'm probably missing some detail).

Reply:
Yes, I agree with what you said:

> `rpcRecHits` is part of reconstruction, which makes me wonder why L1 simulation should depend on reconstruction

The reason was because I plan to eventually include Phase-2 iRPC hits in the EMTF emulator. According to the suggestion from Brieuc Francois, the clusterization of the iRPC hits is done in the `rpcRecHits`, so if I wanted to use the iRPC hits, I have to use `rpcRecHits`. (There is no dedicated L1 producer to do the clusterization). I realized that it was a strange way of doing things, but it did seem to work (at least to me). But now it turned out to be a problem.

I'm not sure what's the best way to move forward (other than to do development in a private CMSSW branch). It seems like the RPC group should develop a L1 producer? But this is not something I could have a say.

Reply:
Could the same EDProducer but with a different module label than `rpcRecHits` work? 

It's the re-use of a module label already used in RECO that causes the specific problem in the unit test.

Reply:
Oh I see. So using the EDProducer itself is not an issue, but I shouldn't be using/re-using the exact `rpcRecHits`. I'll do what you suggested in the future. Thanks!